app/controllers/items_controller.rb

```diff
@@ -1,3 +1,16 @@
 class ItemsController < ApplicationController
-  skip_before_action :authorize, only: :menu
+  skip_before_action :authorize
+
+  def menu
+    @default_branch_id = current_user&.branch_id || params[:branch_id] || Branch.find_by_default(true).id || Branch.first.id
+    @branch = Branch.find_by_id(@default_branch_id)
+
+    @meals = Meal.includes(:branches_meals).where(active: true, branches_meals: { branch_id: @default_branch_id, available: true, active: true })
```

Comment:
Create associations for these conditions.
You can create one for `available_and_active branches_meals` on branch
and then active_meals through `available_and_active branches_meals`
You can call something like
branch.active_meals

Reply:
Done